server/index.js

```diff
@@ -1,8 +1,20 @@
+import seed from './seed.js'
+
 const express = require('express');
 const app = express();
 const port = 5000;
 
```

Comment:
Use bodyparser here to make sure you can parse the data once you receive it.  Also can hook up your static file here to see if your API is working from the client